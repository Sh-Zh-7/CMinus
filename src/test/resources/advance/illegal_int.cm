int main()
{
    int a = 09;
    int b = 0xzz;
}