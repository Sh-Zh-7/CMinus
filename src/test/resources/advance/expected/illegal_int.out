Error type A at Line 3: Illegal octal numbers!
Error type A at Line 4: Illegal hex numbers!
Error type A at Line 5: Illegal decimal numbers!
